apps/i18n/applab/en_us.json

```diff
@@ -511,7 +511,9 @@
   "makerSetupWebSerialConnectToComputer": "Connect the board to your computer using a USB cable.",
   "makerSetupWebSerialConnectToBoardButton": "Press the “Connect to Board” button below.",
   "makerSetupWebSerialWindowConnect": "A window should pop up with a row that includes \"Circuit Playground\" or \"micro:bit\". Select that row, then click \"Connect\".",
-  "makerSetupNoCompatibleDevices":"If you see 'No compatible devices found' for your Circuit Playground Express, follow the [instructions below](#{installInstructionsId}) to install the required Firmata firmware.",
+  "makerSetupWebSerialNotSupported": "WebSerial port is not available in the current browser. To continue, try loading this site from a computer or browser that supports WebSerial.",
+  "makerSetupCpNoCompatibleDevices":"Connecting a Circuit Playground Express and seeing \"No compatible devices found\"?",
```

Comment:
I wonder if there could be an in-between transition phrase. Something like, 'you many not have the required Firmata firmware installed on your board. Instructions below...'

Reply:
Updated!